CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.2.5
```

Comment:
0.2.4+1, since this doesn't add new API surface.

Reply:
Done.